src/modules/app/actions/listen-to-updates.js

```diff
@@ -37,6 +37,15 @@ export function listenToUpdates(history) {
       }
     })
     augur.events.startAugurNodeEventListeners({
+      InitialReportSubmitted: (err, log) => {
```

Comment:
```
  describe('InitialReportSubmitted', () => {
    let testStore
    beforeEach(() => {
      testStore = mockStore.mockStore({})
    })

    it('should dispatch loadMarketsInfo action', () => {
      const mockAugur = {
        events: {
          startAugurNodeEventListeners: () => {
          },
          stopBlockListeners: () => {
          },
          stopAugurNodeEventListeners: () => {
          },
          stopBlockchainEventListeners: () => {
          },
          startBlockListeners: () => {
          },
          nodes: {
            augur: {
              on: () => {
              },
            },
            ethereum: {
              on: () => {
              },
            },
          },
        },
      }

      ReWireModule.__Rewire__('augur', mockAugur)

      const startAugurNodeEventListenersSpy = sinon.spy(mockAugur.events, 'startAugurNodeEventListeners')

      // Setup listeners
      listenToUpdates(mockHistory)(testStore.dispatch, testStore.getState)

      const listenerMap = startAugurNodeEventListenersSpy.firstCall.args[0]
      listenerMap.InitialReportSubmitted(null, {
        market: '0x12345',
      })

      store.getActions()

    })
  })
```

Reply:
John added unit tests, I'll add more for the events I'm adding.